desktop/build.gradle.kts

```diff
@@ -23,9 +23,11 @@ val deployFolder = file("../deploy")
 
 // See https://github.com/libgdx/libgdx/wiki/Starter-classes-and-configuration#common-issues
 // and https://github.com/yairm210/Unciv/issues/5679
-val jvmArgsForMac = listOf("-XstartOnFirstThread", "-Djava.awt.headless=true")
+val customJvmArgs = mutableListOf("-Djava.awt.headless=true")
+if ("mac" in System.getProperty("os.name").toLowerCase())
+    customJvmArgs.add("-XstartOnFirstThread") // Non-standard, only available (and necessary) on Mac.
```

Comment:
Also the second variable is only necessary for Mac

Reply:
Changed.